app/styles/components/_logo.scss

```diff
@@ -1,7 +1,7 @@
-$logo-width: 179px;
-$logo-height: 120px;
+$logo-width: 149px;
+$logo-height: 100px;
 
-$logo-width-mobile: 76px;
+$logo-width-mobile: 75px;
```

Comment:
I know you changed these values to match with the ones on Zeplin, but checking the git history, these values were there from the start. In fact, they match the image dimensions.

They're probably a result of some design changes that weren't reflected on Zeplin, so I'd wait for a feedback from @fcbaila